include/TNetwork.h

```diff
@@ -38,6 +38,7 @@ class TNetwork {
     std::thread mUDPThread;
     std::thread mTCPThread;
 
+    static std::string Hash(const std::string& str);
```

Comment:
This should be named more clearly, such as `HashPassword`, and probably moved to the .cpp file, if it's not called from anywhere else. We generally prefer static globals in the .cpp file to static methods in headers

Reply:
but you already have static methods right below it, and the method just hashes a string it can be anything not just the password. But sure I can rename it to "SHA256" if you want.

Reply:
It hashes a password, so it should be named accordingly, I would say. It doesnt matter how, its just about what intend to do with it

Reply:
To be clear, my suggestion is

- name it HashPassword
- move it to the .cpp file